### ISSUE TITLE
JAWS 2018 - Implicit labels/radios do not work as expected in IE Edge and IE 11

### ISSUE THREAD

**Issue by jasonday** [FreedomScientific/VFO-standards-support#36]
### Implicit labels/radios do not work as expected in IE Edge and IE 11

See test code:
https://codepen.io/jasonday/pen/zppLyG

* Using keyboard enter Implicit label fieldset
* use arrows to navigate between options
    * In IE Edge, each radio is announced "1 of 1"
    * In IE 11
        * No count of number of options is announced (for implicit or explicit)
        * If a selection has been made (implicit), then each option is announced as the first option label. In example demo code, each option is announced as "color 1"

### Expected result

* I expect each option to announce it's index in the count total
* I expect labels to be announced appropriately

**Comment by StommePoes** [comment]
I can reproduce the "no count of radios in group is annonced" in IE11 with NVDA, so perhaps the browser is not offering that.
I don't have Edge except on mobile so can't test :(

Strangely, and I don't recall needing to do this before, but I while I tab into the fieldset I hear the legend, I need to manually turn off Virtual PC cursor off in order to move around the radios (which means I don't hear the legend). Is this new?

**Comment by stevefaulkner** [comment]
In IE11 and JAWS 2018 I cannot reproduce the issue with announcement of incorrect implicit label. JAWS announces 

> "fieldset 1 color 1 radio button not checked"
> "fieldset 1 color 3 radio button not checked"
> "fieldset 1 color 3 radio button not checked"

As far as JAWS not announcing the setsize or position in set, this is because as Mallory suggested, the information is not provided ib IE.

The issue with JAWS and Edge is an Edge bug. Edge does not correctly convey the 
PositionInSet:	SizeOfSet: values correctly in UIAutomation all values = 1

**Comment by stevefaulkner** [comment]
@StommePoes wrote:
Strangely, and I don't recall needing to do this before, but I while I tab into the fieldset I hear the legend, I need to manually turn off Virtual PC cursor off in order to move around the radios (which means I don't hear the legend). Is this new?

**Comment by stevefaulkner** [comment]
Have filed a bug on Edge https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/15383849/

**Comment by jasonday** [comment]
@stevefaulkner the incorrect label announcement happens when selecting an implicit label radio with enter. Space works as expected.

**Comment by StommePoes** [comment]
I could reproduce the repetition of the first label as I went through the others, however this was only after I turned off PC cursor to use my arrow keys.

It might have something to do with it being in a codepen, I often run into weird sh*t on those, even when on "full page view".

**Comment by scottaohara** [comment]
Testing with JAWS 2020, this no longer appears to be an issue with IE11.  
Did not retest with legacy Edge as that is no longer supported by JAWS.  

Closing
